sotodlib/coords/local.py

```diff
@@ -0,0 +1,458 @@
+import numpy as np
+from astropy import units as u
+
+
+'''
+Most of the functions are adapted from pymap3d, to include error calulations 
+'''
+
+def ellipsoid(model='WGS84'):
+
+    '''
```

Comment:
Remove blank line between signature and docstring; use """ for docstring.  (Please do this for all functions.)

Should this function raise an error if called with invalid model?  Or just silently return None?